### ISSUE TITLE
Support for Kogan Smart Kettle 

### ISSUE THREAD

**Issue by EmmaMay66** [AMoo-Miki/homebridge-tuya-lan#192]
Just purchased a kogan smart kettle (it is a tuya device) i was able to obtain the device ID Key and the schema, im unable to get the signiture as when set up as a simple light the logs dont show anything and the device still cant be found, unsure what im doing wrong? is there another way to aquire the signiture? 
Please help in getting support for this device! ive just had a baby and it woudl be super helpful if i could ask siri to boil my kettle ready for bottles. ^_^

Found 1 device:
[
  {
    name: 'Kogan smart kettle',
    id: '41341132bcddc266e0c5',
    key: '87869d43e011b3ae',
    def: {
      schema: '[{"mode":"rw","code":"Power","name":"\\u70e7\\u5f00","property":{"type":"bool"},"iconname":"icon-dp_power","id":1,"type":"obj","desc":""},{"mode":"ro","code":"tempcurrent","name":"Current Temeperature","property":{"unit":"\\u2103","min":0,"max":100,"scale":0,"step":1,"type":"value"},"iconname":"icon-dp_light2","id":5,"type":"obj","desc":""},{"mode":"rw","code":"tem","name":"\\u4fdd\\u6e29","property":{"range":["40","50","60","80","90","currenttemp"],"type":"enum"},"id":102,"type":"obj","desc":"\\u5355\\u4f4d\\u2103"}]',
    },
  },
]

link to purchase = https://www.kogan.com/au/buy/kogan-smarterhome-17l-smart-glass-kettle/

**Comment by ttambo** [comment]
I would also like this! Even if it’s not the full functionality but if it acted like a switch for on and off would be great! @AMoo-Miki

**Comment by freshy84** [comment]
+1 have the same kettle - would love to have it in homebridge

**Comment by xkyle22x** [comment]
> Just purchased a kogan smart kettle (it is a tuya device) i was able to obtain the device ID Key and the schema, im unable to get the signiture as when set up as a simple light the logs dont show anything and the device still cant be found, unsure what im doing wrong? is there another way to aquire the signiture? 
> 
> Please help in getting support for this device! ive just had a baby and it woudl be super helpful if i could ask siri to boil my kettle ready for bottles. ^_^
> 
> 
> 
> Found 1 device:
> 
> [
> 
>   {
> 
>     name: 'Kogan smart kettle',
> 
>     id: '41341132bcddc266e0c5',
> 
>     key: '87869d43e011b3ae',
> 
>     def: {
> 
>       schema: '[{"mode":"rw","code":"Power","name":"\\u70e7\\u5f00","property":{"type":"bool"},"iconname":"icon-dp_power","id":1,"type":"obj","desc":""},{"mode":"ro","code":"tempcurrent","name":"Current Temeperature","property":{"unit":"\\u2103","min":0,"max":100,"scale":0,"step":1,"type":"value"},"iconname":"icon-dp_light2","id":5,"type":"obj","desc":""},{"mode":"rw","code":"tem","name":"\\u4fdd\\u6e29","property":{"range":["40","50","60","80","90","currenttemp"],"type":"enum"},"id":102,"type":"obj","desc":"\\u5355\\u4f4d\\u2103"}]',
> 
>     },
> 
>   },
> 
> ]
> 
> 
> 
> 
> 
> link to purchase = https://www.kogan.com/au/buy/kogan-smarterhome-17l-smart-glass-kettle/

Any luck with this. I'm thinking about getting one and trying to work it out.

**Comment by ttambo** [comment]
> > Just purchased a kogan smart kettle (it is a tuya device) i was able to obtain the device ID Key and the schema, im unable to get the signiture as when set up as a simple light the logs dont show anything and the device still cant be found, unsure what im doing wrong? is there another way to aquire the signiture?
> > Please help in getting support for this device! ive just had a baby and it woudl be super helpful if i could ask siri to boil my kettle ready for bottles. ^_^
> > Found 1 device:
> > [
> > {
> > ```
> > name: 'Kogan smart kettle',
> > 
> > id: '41341132bcddc266e0c5',
> > 
> > key: '87869d43e011b3ae',
> > 
> > def: {
> > 
> >   schema: '[{"mode":"rw","code":"Power","name":"\\u70e7\\u5f00","property":{"type":"bool"},"iconname":"icon-dp_power","id":1,"type":"obj","desc":""},{"mode":"ro","code":"tempcurrent","name":"Current Temeperature","property":{"unit":"\\u2103","min":0,"max":100,"scale":0,"step":1,"type":"value"},"iconname":"icon-dp_light2","id":5,"type":"obj","desc":""},{"mode":"rw","code":"tem","name":"\\u4fdd\\u6e29","property":{"range":["40","50","60","80","90","currenttemp"],"type":"enum"},"id":102,"type":"obj","desc":"\\u5355\\u4f4d\\u2103"}]',
> > 
> > },
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > },
> > ]
> > link to purchase = https://www.kogan.com/au/buy/kogan-smarterhome-17l-smart-glass-kettle/
> 
> Any luck with this. I'm thinking about getting one and trying to work it out.

Yes please, this would be the last device I need to rely on Alexa for!
